### ISSUE TITLE
Selection or creation of an organization when installing an app is not taken into account

### ISSUE THREAD

**Issue by bobeal** [ozwillo/ozwillo-portal#324]
As reported by @xmasclaux : 

> For an account who are admin of at least 2 organizations, 
> when I install a new app it'll be suscribed for the first organization of the form field ("vos organisations") > even if I choose another one or create a new one.
> Tested in prod with differents accounts and apps : flux demat, simplewiki

**Comment by bobeal** [comment]
See https://github.com/ozwillo/ozwillo-issues/issues/5

**Comment by bobeal** [comment]
I can't believe I'm understanding this line : https://github.com/ozwillo/ozwillo-portal/blob/master/portal-parent/ozwillo-portal-front/src/main/resources/public/js/store/store-install-app.jsx.js#L872 ...

**Comment by schambon** [comment]
With the current fix in dev: if the user wants to select the first organization, she has to open the select box, select the second one, then select the first one again.
(If the user is admin of only one org, she's screwed.)

Basically the first item in the list isn't selected by default. This can be very annoying or even blocking if one only has one org (which is the most common use case)

**Comment by bobeal** [comment]
That somehow explains the uncommented hack I removed in toggleType (which was also causing a bug when toggling between new and existing organizations).

To be honest, I've refactored this part of the process while working on the integration of the new design (which will be available next week) and I don't have time to dig into this problem this week. So I would prefer let it like that until the new design is available.
